apps/meteor/tests/end-to-end/api/rooms.ts

```diff
@@ -3647,4 +3648,179 @@ describe('[Rooms]', () => {
 				});
 		});
 	});
+
+	describe('[/rooms.membersOrderedByRole]', () => {
```

Comment:
Add test cases for:
- Trying to fetch a private room by a user not part of room
- Trying to fetch a public room by a user not part of room
- The same 2, but with users that are part of the rooms
- Trying to fetch a private channel inside a public team for someone not part of the room
- Trying to fetch a public channel inside a private team for someone not part of the room
- The same 2, but with users that are part of the rooms

Just to ensure we don't show more data than what we should 🙈